examples/fonts-custom/docs/README.md

```diff
@@ -0,0 +1,196 @@
+# Using a custom font
+
+This example uses the Noto Serif and Noto Mono font families to show
+how custom fonts can be used. These families were chosen because the 
+[Noto fonts are well documented] and they are available under the 
+[Open Font License].
+
+[Noto fonts are well documented]: https://notofonts.github.io/noto-docs/
+[Open Font License]: https://scripts.sil.org/ofl
+
+In this example, the fonts are loaded from a CDN (that is not Google)
+to keep the example itself small and allow you to use it as a template
+for your own site, without assuming what fonts you might want to use.
+We also describe how to use custom fonts that are hosted on your own
+server, see below.
+
+Here we have some text in:
+
+- Noto Serif
+- **Noto Serif bold**
+- *Noto Serif italic*
+- __*Noto Serif bold italic*__
+- `Noto Mono`
+- __`Noto Mono bold`__ 
+
+## How it works
+
+In `mkdocs.yml`, turn off the auto-loading of fonts from Google Fonts
+and add an extra CSS stylesheet to configure your custom fonts.
+
+```yaml
+theme:
+  font: false
+
+extra_css:
+  - assets/stylesheets/extra.css
+```
+
+Then, [configure custom fonts] hosted either on a CDN or on your own
+server, depending on your needs. We demonstrate this here with the 
+Noto Sans and Noto Mono font families.
+
+[configure custom fonts]: https://squidfunk.github.io/mkdocs-material/setup/changing-the-fonts/#additional-fonts
+
+### Using a CDN's CSS
+
+If you are using a CDN to load your fonts, you may want to use CSS
+that the operators of the CDN are providing for defining the font
+faces, instead of writing your own. You still need to configure the
+fonts for Material to use, though:
+
+=== "`mkdocs.yml`"
+
+    ```yaml
+    extra_css:
+      - https://fonts.cdnfonts.com/css/noto-serif
+      - https://fonts.cdnfonts.com/css/noto-mono
+      - assets/stylesheets/extra.css
+    ```
+
+=== "`extra.css`"
+
+    ```css
+    :root {
+      --md-text-font: "Noto Serif";
+      --md-code-font: "Noto Mono";
+    }
+    ```
+
+### Using a CDN with your own CSS
```

Comment:
I'm not sure we need this example – it's actually only the first step for self hosting, as the font files are still hosted on the CDN. I think I would drop this one and recommend the privacy plugin which takes care of all of this. With the privacy plugin, you get self hosting with the first example and a single line of config. It will be released when we reach the next funding goal, which is just around the corner.